libraries/abstractions/common_io/test/test_iot_i2c.c

```diff
@@ -107,18 +102,6 @@ TEST_GROUP( TEST_IOT_I2C );
  */
 TEST_SETUP( TEST_IOT_I2C )
 {
-    IotI2CHandle_t xGlobalI2CHandle = gIotI2cHandle[ testIotI2C_INSTANCE ];
-    int32_t lRetVal;
-
-    /* Close global handle it was opened. */
-    if( xGlobalI2CHandle != NULL )
-    {
-        lRetVal = iot_i2c_close( xGlobalI2CHandle );
-        TEST_ASSERT_EQUAL( IOT_I2C_SUCCESS, lRetVal );
-
-        gIotI2cHandle[ testIotI2C_INSTANCE ] = NULL;
-    }
-
```

Comment:
Let me confirm if removing this OK.

Reply:
I removed it based on the assumption that before running a test, any resources once expects to test should be released and free. I don't really see the use-case where the tests need to be designed to potentially clean up after another part of the tests setup.